Add len() variants for ustack and kstack (#3769)
bpftrace/bpftrace: CHANGELOG.md

  - [#3678](https://github.com/bpftrace/bpftrace/pull/3678)
- Change max_strlen default from 64 to 1024
  - [#3713](https://github.com/bpftrace/bpftrace/pull/3713)

> - The `len` function now accepts `ustack` and `kstack` values as its argument

Reviewer: nit
```suggestion
- `len` now also accepts `ustack` and `kstack` as arguments
```

Also move this under `added`
